### ISSUE TITLE
Please provide docker

### ISSUE THREAD

**Issue by utkutpcgl** [Xianpeng919/MonoCon#31]
Thanks for your contribution! 

Suggestion: The installations are cumbersome (probably not supported/incompatible versions exist), so a docker would be much easier.

**Comment by aminullah6264** [comment]
I agree, seems like new versions mmcv, mmdetection and mmdetection3d are not compatible, I tried to compile several version but couldn't make it run.  timm no longer working with with torch version less than 1.7 etc.

**Comment by Lpy15540250240** [comment]
same to me , be quick!
